### PR TITLE
Changed FreeLicenceLookup from raw sql to orm.

### PR DESCRIPTION
**Pull request by xliiv**
Used `ifnull` as a custom aggregator function + added tests.
Raw sql gave bugs.

### CODE REVIEW

**Comment by ar4s** [comment]
:ok:

File: src/ralph_assets/models.py [hunk 1]
```diff
@@ -130,45 +132,37 @@ def format_item_display(self, obj):
         return item
 
 
+class MySum(SqlAggregate):
```
Comment (zefciu) [inline src/ralph_assets/models.py]: Not quite descriptive name. Maybe a little docstring would help?

File: src/ralph_assets/models.py [hunk 2]
```diff
@@ -130,45 +132,37 @@ def format_item_display(self, obj):
...
+    sql_function = 'SUM'
+    sql_template = 'IFNULL(%(function)s(%(field)s), 0)'
+
+
+class NewSum(Aggregate):
```
Comment (zefciu) [inline src/ralph_assets/models.py]: ditto

**Comment by andrzej-jankowski** [comment]
:ok:

**Pull request merged**